### DISCUSSION TITLE
Layer-like Level Panel

### DISCUSSION THREAD

**Discussion by VinnNo** [tahoma2d/tahoma2d#1638] [Ideas]
I think it would be VERY helpful to have a layer panel that works like what you'd see in traditional painting programs.

At the very top, it would have layer blending modes for the select level.
Directly under it, it would have the opacity for the selected level.
Under this is render visibility, stage visibility, level lock, new level, masking to mask the select level with the level underneath.
The is the levels themselves. with a fairly standard layout.
Ideally, subscenes could be collapsed and expanded here like layers do inside of groups and folders.

Below is a mockup of what I'd love to see in this kind of panel.

![TahomaLayerMockup](https://github.com/user-attachments/assets/d3a991d3-5748-4d23-8698-65f1529bde17)

I realize this can be done via the FX room, but it is a lot of node wrangling to have a level set to multiply as you have to plug in every level that you want this to multiply on in to the Layer Blend node. It's so much of a time-sink that we handle this in another program, but we'd love to have less layers coming out of Tahoma and simply have the blend modes apply prior to render.

In instances like layer blending, I think it makes sense to have it take the selected level and apply it to all levels below it rather than having to do this every time.

![image](https://github.com/user-attachments/assets/144b1c44-e785-4c0f-abf2-a6a17b1fcd3e)

If I could be pointed at the specific source files that would be needed for the implementation of this to poke around myself, that would also be super-duper helpful!

Thanks for reading!
-Vinn

**Comment by artisteacher** [comment]
If the blending modes could be real-time, that would be amazing. There's already an opacity slider for each layer, but having it exposed means less clicking. This would add more vertical space to the top of the Timeline, which is tough on small screen sizes.
